File: Q2.java
import java.util.Scanner;

public class Q2 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        float numero;
        System.out.print("Digite um número: ");
        numero = leia.nextFloat();
        if (numero > 10) 
        {
            System.out.println("É MAIOR QUE 10!");
        } 
        else 
        {
            System.out.println("NÃO É MAIOR QUE 10!");
        }
    }
}

File: Q2.cs
using System;

namespace Q2
{
    class Program
    {
        static void Main(string[] args)
        {
            float número;
            Console.Write("Digite um número: ");
            número = float.Parse(Console.ReadLine());
            if (número > 10)
            {
                Console.WriteLine("É MAIOR QUE 10!");
            }
            else
            {
                Console.WriteLine("NÃO É MAIOR QUE 10!");
            }
        }
    }
}